test/test_qwerty_mlir/Qwerty/IR/verifier.mlir

```diff
@@ -0,0 +1,119 @@
+// RUN: qwerty-opt %s --split-input-file --verify-diagnostics
+
+module {
+  qwerty.func @sciff_iff_0[]() irrev-> !qwerty<bitbundle[1]> {
```

Comment:
It might help to rename this to something like `@linearity_scf_if_result_used_twice` or something